packages/ckeditor5-bookmark/tests/bookmarkediting.js

```diff
@@ -174,6 +224,19 @@ describe( 'BookmarkEditing', () => {
 			);
 		} );
 
+		it( 'should properly downcast bookmark with text with space after', () => {
+			setModelData( model, '<paragraph><bookmark bookmarkId="foo"></bookmark> text</paragraph>' );
+
+			expect( getViewData( view, { withoutSelection: true } ) ).to.equal(
+				'<p>' +
+					'<a class="ck-bookmark ck-widget" contenteditable="false" id="foo">' +
+						'<span class="ck-bookmark__icon"></span>' +
+					'</a>' +
+					'text' +
```

Comment:
The space must be kept here also.

Reply:
https://github.com/ckeditor/ckeditor5/pull/17073#discussion_r1756213693